src/common/envalid.ts

```diff
@@ -20,32 +30,29 @@ const cliEnvSpec = {
   OTOMI_DEV: bool({ default: false }),
   OTOMI_IN_TERMINAL: bool({ default: true }),
   STATIC_COLORS: bool({ default: false }),
-  TESTING: bool({ default: false }),
   TRACE: bool({ default: false }),
   VERBOSITY: num({ desc: 'The verbosity level', default: 1 }),
   VALUES_INPUT: str({ desc: 'The chart values.yaml file', default: undefined }),
 }
 
-export const cleanEnvironment = (spec: Record<string, any> = cliEnvSpec): Record<string, any> => {
+export function cleanEnv<T>(
+  spec: { [K in keyof T]: ValidatorSpec<T[K]> },
+  options?: CleanOptions<T>,
+): Readonly<T & CleanedEnvAccessors> {
   let pEnv: any = process.env
-  // load local .env if we have it, for devs
-  let path = `${process.cwd()}/.env`
-  if (!process.env.TESTING && existsSync(path)) {
-    const result = config({ path })
-    if (result.error) console.error(result.error)
-    pEnv = { ...result.parsed, ...pEnv }
-  }
-  path = `${pEnv.ENV_DIR}/.secrets`
-  if (existsSync(path)) {
-    const result = config({ path })
-    if (result.error) console.error(result.error)
-    pEnv = { ...result.parsed, ...pEnv }
-  }
-  return cleanEnv(pEnv, spec)
+  // load local .env and $ENV_DIR/.secrets if we have it
+  ;[`${process.cwd()}/.env`, `${pEnv.ENV_DIR}/.secrets`].forEach((path) => {
```

Comment:
what does the `;` stand for?

Reply:
ask eslint...but I assume it is for correct compiling to have separation between lines